ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/svarinn/SvarInnClient.java

```diff
@@ -0,0 +1,37 @@
+package no.difi.meldingsutveksling.ks.svarinn;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class SvarInnClient {
+    public static final MediaType APPLICATION_ZIP = MediaType.parseMediaType("application/zip;charset=UTF-8");
+    private RestTemplate restTemplate;
+
+    public SvarInnClient(RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public List<Forsendelse> checkForNewMessages() {
+        final List<Forsendelse> forsendelser = Arrays.asList(restTemplate.getForObject("/mottaker/hentNyeForsendelser", Forsendelse[].class));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "forsendelser". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1488)